stdlib/asm/word.masm

```diff
@@ -0,0 +1,228 @@
+#! Reverses order of the first four elements on the stack
+#!
+#! Inputs:  [a, b, c, d, ...]
+#! Outputs: [d, c, b, a, ...]
+#!
+#! Cycles: 3
+export.reverse
+	movdn.3
+	swap.2
```

Comment:
Probably the 8 spaces tab was used here? I would make it 4 spaces to be consistent with other procedures.